### PR TITLE
Update input types and vars to support runtime config

### PR DESCRIPTION
**Pull request by uri-weisman**
## What does this PR do?

1. Update the `dataYaml` var to `runtimeCfg`
2. Modify Cloudbeat's input types to make the Kibana plugin benchmark agnostic.

## Checklist

- [ ] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [ ] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [ ] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Author's Checklist

- [ ]

## Related issues

-

### CODE REVIEW

**Review by eyalkraft — COMMENTED** [review]
let's
```
conditions:
  kibana.version: "^8.4.0"
  ```
  
  So this won't break the current 8.4 release

**Review by eyalkraft — APPROVED** [review]
Merge with the proper changes in Kibana & cloudbeat

File: packages/cloud_security_posture/manifest.yml [hunk 1]
```diff
@@ -43,16 +43,16 @@ policy_templates:
...
-      - type: cloudbeat/eks
+      - type: cloudbeat/cis_eks
         title: Enable Amazon EKS Benchmark
         description: Collecting findings
 vars:
-  - name: dataYaml
+  - name: runtimeCfg
     type: yaml
-    title: Rules Activation Yaml
+    title: Rules Activation
```
Comment (kfirpeled) [inline packages/cloud_security_posture/manifest.yml:55]: consider changing the title to `List of activated rule ids`

**Review by kfirpeled — APPROVED** [review]

File: packages/cloud_security_posture/manifest.yml [hunk 2]
```diff
@@ -1,15 +1,16 @@
...
+release: beta
 license: basic
 description: "Check Kubernetes cluster compliance with the Kubernetes CIS benchmark."
 type: integration
 categories:
   - containers
   - kubernetes
 conditions:
-  kibana.version: "^8.5.0"
+  kibana.version: "^8.4.0"
```
Comment (eyalkraft) [inline packages/cloud_security_posture/manifest.yml:13]: This has to be 
```suggestion
  kibana.version: "^8.5.0"
```
no?

Comment (uri-weisman) [reply packages/cloud_security_posture/manifest.yml]: Sure, overrode by upstream merge.

File: packages/cloud_security_posture/changelog.yml [hunk 3]
```diff
@@ -1,9 +1,17 @@
...
       type: enhancement
       link: https://github.com/elastic/integrations/pull/3968
+- version: "0.0.26"
+  changes:
+    - description: Version bump
+      type: enhancement
+      link: https://github.com/elastic/integrations/pull/3997
+    - description: Updates to KSPM Integration README
+      type: enhancement
+      link: https://github.com/elastic/integrations/pull/3964
```
Comment (oren-zohar) [inline packages/cloud_security_posture/changelog.yml:14]: Since 0.0.26 wasn't promoted yet we can unify it under it, no?

**Review by oren-zohar — CHANGES_REQUESTED** [review]

File: packages/cloud_security_posture/changelog.yml (hunk 3 above)
Comment (uri-weisman) [reply packages/cloud_security_posture/changelog.yml:14]: This pr introduces breaking changes so we want to promote version 0.26 to SNAPSHOT and only then merge this commit.
The Kibana version condition changed to ^8.5.0.

**Review by oren-zohar — APPROVED** [review]

**Pull request merged**